### ISSUE TITLE
[FEATURE] Media does not play automatically

### ISSUE THREAD

**Issue by duckmonkey** [thanksmister/wallpanel-android#293]
I'm using Home Assistant and browser_mod with wallpanel to pop up a stream of my front door camera (using more-info) when the doorbell is pressed.  However, in wallpanel, the stream loads, but I am left with a "play" button in the media control and it does not play until this is pressed,  I would like it to play automatically.

I assume that it requires this option from WebSettings:

[WebSettings#setMediaPlaybackRequiresUserGesture(boolean)](https://developer.android.com/reference/android/webkit/WebSettings#setMediaPlaybackRequiresUserGesture(boolean))

**Comment by thanksmister** [comment]
This could only be handled by WebView component or if we use an alternative method for playing video in the WebView.

**Comment by oramirite** [comment]
I don't think this is a WallPanel problem. I have this same issue with a bunch of other setups, including launching the popup on my Laptop and launching it in the HA Companion App itself on a wall mounted Amazon Fire HD 8. I primarily have this issue when using WebRTC. One interesting note is that I've never had the problem when launching this popup on a device itself via a buttonpress - it's always when I'm launching the popup via automation remotely.

My guess is that it's something to do with how browser_mod handles this process, as it's the only common denominator.

**Comment by pissten** [comment]
Hi,
I'm using browser_mod ( home-assistant) to popup a camera screen when motion is detected.
I managed to get the popup to show the camera feed but i have to manually press the “play” button to get the feed going.
I've got the feeling that the auto-play function in Fully Kiosk Browser, takes care of this issue, but is it possible to do something similar in Wallpanel?
